tests/nodeos_snapshot_diff_test.py

```diff
@@ -152,10 +156,9 @@ def waitForBlock(node, blockNum, blockType=BlockType.head, timeout=None, reportI
     assert steadyStateAvg>=minRequiredTransactions, "Expected to at least receive %s transactions per block, but only getting %s" % (minRequiredTransactions, steadyStateAvg)
 
     Print("Create snapshot")
-    ret = nodeSnap.createSnapshot()
+    ret = nodeSnap.scheduleSnapshot()
```

Comment:
I think this is probably fine. Would have been better to enhance this test to do both, but not a huge deal.

Reply:
Done, now 3 snapshots are being generated and compared to each other